recipe/meta.yaml

```diff
@@ -1,23 +1,22 @@
-{% set name = "distarray" %}
 {% set version = "2.12.2" %}
 
 package:
-  name: {{ name|lower }}
+  name: distarray
   version: {{ version }}
 
 source:
-  url: https://github.com/UV-CDAT/distarray/archive/2.12.2.tar.gz
+  url: https://github.com/UV-CDAT/distarray/archive/{{ version }}.tar.gz
   sha256: ca6ee7610b8e8309cec47121b2454d8d318bfe37b6db3b950c65b1316035ba6c
 
 build:
-  number: 0
-  skip: True  # [win]
+  number: 1
+  noarch: python
```

Comment:
@doutriaux1 I could not find a compiled extension in the code so I'm testing this. Please let me know if this is wrong and, if so, point me to the compiled part so we can use the `jinja` compile syntax here instead.

Reply:
@ocefpaf There is no compiled extension, but don't you need to do a `python setup.py install` for `conda build` to create the right tarball?

Reply:
We have that in the line below with `pip`.